Dockerfile.test

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.2.4
```

Comment:
No, I can't think of any. There's a decent set of specs and cukes, so I'd expect a failure somewhere in them if this was going to cause a problem